### ISSUE TITLE
Appveyor builds

### ISSUE THREAD

**Issue by jayvdb** [brechtm/citeproc-py#36]
Do you mind if I add an Appveyor build script, either as the default `appveyor.yml` or using `.appveyor.yml` ?

http://ci.appveyor.com/

Personally I rather that it downloads the build scripts from the https://github.com/ogrisel/python-appveyor-demo project, similar to how it is done here:

https://github.com/wikimedia/pywikibot-core/blob/master/.appveyor.yml

Also, I see you recently added https://coverage.io.  Would you mind terribly if we switched to https://codecov.io , as it groups jobs by commit, merging Appveyor and Travis jobs into the same coverage report.

**Comment by brechtm** [comment]
Yes, adding Appveyor CI would be nice. I'd prefer `.appveyor.yml` myself.

Do we really need to depend on python-appveyor-demo? This seems to be dedicated to binary builds...
Appveyor does not have Python 3.2 though, but that seems to be easy enough to install: https://github.com/JetBrains/teamcity-messages/blob/master/appveyor.yml

I'm fine with switching to codecov.io. But let's leave coverage.io enables as long as codecov.io is not yet configured and working.

**Comment by brechtm** [comment]
If possible, try running tox as with Travis CI to avoid duplication. But it's probably a good idea to get things running first without that complication.

**Comment by jayvdb** [comment]
python-appveyor-demo provides versions of Python which are not pre-installed on Appveyor build machines, and helps configure everything to install packages that require a compiler.
IMO the pre-installed versions are likely sufficient for this projects testing needs.
I'll try using tox, and we can re-evaluate as we go.
